Flagsmith.Engine/Segment/Models/SegmentModel.cs

```diff
@@ -12,8 +12,8 @@ public class SegmentModel
         [JsonProperty("name")]
         public string Name { get; set; }
         [JsonProperty("rules")]
-        public List<SegmentRuleModel> Rules { get; set; }
+        public List<SegmentRuleModel> Rules { get; set; } = new List<SegmentRuleModel>();
         [JsonProperty("feature_states")]
-        public List<FeatureStateModel> FeatureStates { get; set; }
+        public List<FeatureStateModel> FeatureStates { get; set; } = new List<FeatureStateModel>();
```

Comment:
Ensure that default lists are instantiated here to prevent NullPointerExceptions